tests/packages/drizzle/http.test.ts

```diff
@@ -0,0 +1,19 @@
+import { expect, test } from 'vitest';
+import { neon } from '../../../dist/npm';
+import { drizzle } from 'drizzle-orm/neon-http';
+import { sql } from 'drizzle-orm';
+
+// note that these tests rely on this line in package.json's "devDependencies":
+// "@neondatabase/serverless": "file:dist/npm"
+
+const DATABASE_URL = process.env.VITE_NEON_DB_URL!;
+
+test('basic query using drizzle-orm with https fetch', async () => {
```

Comment:
WDYT about setting up a project with Neon Authorize enabled and test with passing the auth token using the `authenticated` role?

Reply:
Yeah, that's a good idea. It will also teach me to use Neon Authorize ...

Reply:
OK, that's done: https://github.com/neondatabase/serverless/blob/jawj/adding-tests/tests/cli/httpAuthorize.test.ts